### PR TITLE
Alemata.angular2 mfa

### PR DESCRIPTION
**Pull request by alemata**

### CODE REVIEW

**Comment by auth0build** [comment]
Can one of the admins verify this patch?

**Comment by alemata** [comment]
@flaviogranero could you review please? It has also react code, sorry

**Comment by flaviogranero** [comment]
👍 @chenkie @mpaktiti

**Comment by alemata** [comment]
@chenkie first merge #1856

**Comment by mpaktiti** [comment]
Merge sequence:
- https://github.com/auth0/docs/pull/1856
- https://github.com/auth0/docs/pull/1857
- https://github.com/auth0/docs/pull/1858

File: articles/client-platforms/_includes/_mfa-login.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+There is no need to update the code we have created in [Login](/docs/quickstart/spa/react/01-login) tutorial. As soon you try to sign in, you should be prompted to setup a second factor authenticator application (Auth0 Guardian is the default).
+
+![guardian screen](/media/articles/mfa/choose-mfa.png)
+
+For detailed instructions on using guardian to authenticate, please refer to [How to Use the Guardian App](/multifactor-authentication/guardian/user-guide)
```
Comment (mpaktiti) [inline articles/client-platforms/_includes/_mfa-login.md]: @alemata missing a fullstop at the end of the sentence.

File: articles/client-platforms/angular2/09-mfa.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
+---
+title: MFA
+description: This tutorial will show you how to add Multifactor Authentication to your Angular2 app with auth0.
+---
+
+<%= include('../../_includes/_github', {
+  link: 'https://github.com/auth0-angularjs2-systemjs-sample/tree/master/09-MFA'
```
Comment (mpaktiti) [inline articles/client-platforms/angular2/09-mfa.md]: @alemata The URL is `https://github.com/auth0-samples/auth0-angularjs2-systemjs-sample/tree/master/09-MFA`

**Comment by alemata** [comment]
@mpaktiti done!

**Pull request merged**